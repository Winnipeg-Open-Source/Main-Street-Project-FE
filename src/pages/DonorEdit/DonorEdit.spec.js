import React from 'react';
import renderWithThemeAndRouter from 'tests/helpers/renderWithThemeAndRouter';
import DonorEditPage from './DonorEdit';

describe ('DonorEditPage', () => {
    it ('renders without crashing', () => {
        const { asFragment } = renderWithThemeAndRouter(<DonorEditPage />);

        expect(asFragment()).toMatchSnapshot();
    });
});
