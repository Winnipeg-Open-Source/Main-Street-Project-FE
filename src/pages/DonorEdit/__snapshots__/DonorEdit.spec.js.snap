// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonorEditPage renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="sc-pkfPC haHNKR sc-psDhf iAENMo"
  >
    <div
      class="sc-AxirZ sc-AxhCb jTGKnC"
      width="1"
    >
      <div
        class="sc-AxirZ sc-AxhCb iEMUrT"
        width="1"
      >
        <div
          class="sc-AxhUy sc-qQxFs hINyDI"
          font-size="3"
        >
          Donor Edit
        </div>
      </div>
      Content
    </div>
  </div>
</DocumentFragment>
`;
